pkg/mdformatter/linktransformer/link.go

```diff
@@ -304,6 +315,26 @@ func (v *validator) visit(filepath string, dest string, lineNumbers string) {
 	}
 }
 
+// checkEmail checks email structure and domain.
+func checkEmail(email string) bool {
```

Comment:
```suggestion
func isValidEmail(email string) bool {
```

Check is not very verbose  what exactly we are checking for